network/src/transport.rs

```diff
@@ -118,22 +118,6 @@ impl ConnectionMetadata {
             application_protocols,
         }
     }
-
```

Comment:
this is a good commit

Reply:
Yeah, it is good to get rid of this anti-pattern.  It's something I think we need to do as a whole sweep of the networking code